modules/quick-start-components.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * web_console/creating-quick-start-tutorials.adoc
+
+[id="quick-start-components_{context}"]
+= Quick start components
+
+A quick start consists of these sections.
+
+* *Card*: The catalog tile that provides the quick start’s basic information, including title, description, time commitment, and completion status.
```

Comment:
Should we include `icon` since icon is also basic information about consolequickstarts?

Reply:
Good idea! We will be expanding this content post-4.7 and will add that information in then.